docs/control_center_docs/control_center_start_guide.rst

```diff
@@ -705,47 +705,36 @@ Once clicked, a File Explorer will open, and you can select a name and location
 
 Adding Resource Files
 =============================
-Occasionally, it is necessary to manually add module resource files to the Control Center. These will be provided by a member of the Vertiq team. 
-The process for integrating the files into the Control Center is described below for each supported operating system.
+Occasionally, it is necessary to manually add module resource files to the Control Center. These will be provided by a member of the Vertiq team as a .zip file. 
+Store this .zip file in a location that is easily accessible. The process for integrating the files into the Control Center is described below.
 
-Windows
-----------------
-1. Use the File Explorer to locate the Control Center's installation. By default this is Program Files (x86)/IQ Control Center
-2. Open the installed folder, and you'll see
+.. note::
+    The following instructions apply to all supported operating systems: Windows, Mac, and Linux (Ubunutu). The first step requires the Control Center application to be opened with administrator privileges.
+    On Mac and Linux, you must use the 'sudo' command in the terminal to run the application.
 
-.. image:: ../_static/control_center_pics/windows/windows_installed_folder.png
+    .. image:: ../_static/control_center_pics/linux/linux_sudo_command.png
 
-3. Enter the IQ Control Center folder, and you’ll see the Resources folder
+1. Use the File Explorer to locate the Control Center's installation. By default, the application is located in Program Files (x86)/IQ Control Center/IQ Control Center/IQ Control Center.exe
+2. Right click the executable and select 'Run as administrator'.
```

Comment:
@Vertiq-Ben Step 1 and 2 here are Windows specific I believe. Whereas the stuff for launching on Mac or Linux are in a note above these instructions. These numbered steps are not platform agnostic. Maybe it would be better to have a step that is "Run the Control Center as Administrator", and then as subpoints to that say you could have the instructions for Mac and Linux, and the instructions for Windows